File: Examples/CSharp/Load-Save-Convert/ExportToXPS.cs
using System.IO;
using Aspose.Diagram;
namespace Aspose.Diagram.Examples.CSharp.Diagrams
{
    public class ExportToXPS
    {
        public static void Run()
        {
            // ExStart:ExportToXPS
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Diagrams();

            // Open a VSD file
            Diagram diagram = new Diagram(dataDir + "LayOutShapesInCompactTreeStyle.vdx");

            // Save diagram to an XPS format
            diagram.Save(dataDir + "Output.xps", SaveFileFormat.XPS);
            // ExEnd:ExportToXPS
        }
    }
}

File: Examples/VisualBasic/Load-Save-Convert/ExportToXPS.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Diagram

Namespace Diagrams
    Public Class ExportToXPS
        Public Shared Sub Run()
            ' ExStart:ExportToXPS
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_LoadSaveConvert()

            ' Open VSD diagram
            Dim diagram As New Diagram(dataDir & "ExportToXPS.vsd")

            ' Save diagram to XPS format
            diagram.Save(dataDir & "Output.xps", SaveFileFormat.XPS)
            ' ExEnd:ExportToXPS
        End Sub
    End Class
End Namespace